apps/get_it/lib/app/app_startup/app_startup_model.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:core/models/app_state/app_startup_state.dart';
+import 'package:core/persistence/data_store.dart';
+import 'package:flutter/foundation.dart';
+
+class AppStartupModel extends ValueNotifier<AppStartupState> {
+  AppStartupModel({required this.dataStore})
+      : super(const AppStartupState.initializing()) {
+    init();
+  }
+  final DataStore dataStore;
+  late StreamSubscription _subscription;
+
+  @override
+  void dispose() {
+    _subscription.cancel();
+    super.dispose();
+  }
+
+  void init() {
+    _subscription = dataStore.profilesData().listen((profilesData) {
+      if (profilesData.selectedId == null || profilesData.profiles.isEmpty) {
+        value = AppStartupState.needsProfile();
+      } else {
+        value = AppStartupState.profileLoaded(profilesData);
+      }
```

Comment:
Various pages in the app depend on the value of `profilesData` (which is emitted by the `Stream`) above.

The easiest way to read this would be with `get_it_mixin`:

```dart
watch<AppStartupModel, AppStartupState>()
```

However, because there are multiple listeners, this error is thrown:

```
This Object is already watched by get_it_mixin
```

What's the right way of doing this?